replica/database.hh

```diff
@@ -1081,6 +1081,10 @@ public:
         return _tombstone_gc_enabled;
     }
 
+    bool tombstone_gc_enabled(locator::tablet_id tid) const {
+        return _sg_manager->tombstone_gc_enabled(tid);
+    }
```

Comment:
I think we combine _tombstone_gc_enabled and _sg_manager->tombstone_gc_enabled(tid) here instead, so it's hidden from compaction_group::table_state, which will only call table::tombstone_gc_enabled(locator::tablet_id).